### PR TITLE
Feat/new sell modal

### LINKED ISSUE #523: Buy/Sell modal pricing data
**"Sell" modal**

Figma: https://www.figma.com/file/7IAQbLyzapnrow9sIcO77P/App?node-id=1968%3A91403&t=9PacbvlULzq2HkAI-0

By default, Trend, Momentum, Steady ETH and Steady BTC strategies will withdraw proportionally current active assets such as ETH, BTC, and USDC.

We want to show next parameters:
ETH Weight
BTC Weight
USDC Weigh
Estimated USD
Image

**"Buy" modal**
Image
We want to show next parameters:
Tokens
Estimated USD

### PR DESCRIPTION
**Pull request by codingki**
Fixes #523

## Description

View withdrawable asset distribution on sell modal only on ETH BTC Mom and Trend

### CODE REVIEW

File: src/components/_modals/WithdrawModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import { SecondaryButton } from "components/_buttons/SecondaryButton"
+import { strategyPageContentData } from "data/strategyPageContentData"
+import { useRouter } from "next/router"
+import { ExchangeTab } from "./ExchangeTab"
+import { SommelierTab } from "./SommelierTab"
+
+type WithdrawModalProps = Pick<ModalProps, "isOpen" | "onClose">
+
+export const WithdrawModal: VFC<WithdrawModalProps> = (props) => {
+  // eslint-disable-next-line react/display-name
```
Comment (joshuanatanielnm) [inline src/components/_modals/WithdrawModal/index.tsx:28]: why disable eslint?

**Review by joshuanatanielnm — APPROVED** [review]
LGTM, just have some question about `ts-ignore` and `disable es-lint`

File: src/components/_modals/WithdrawModal/SommelierTab.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+type WithdrawModalProps = Pick<ModalProps, "isOpen" | "onClose">
+
+export const SommelierTab: VFC<WithdrawModalProps> = ({
+  isOpen,
+  onClose,
+}) => {
+  return (
+    <>
+      <WithdrawForm onClose={onClose} />
+    </>
```
Comment (xanderjl) [inline src/components/_modals/WithdrawModal/SommelierTab.tsx:14]: Is this fragment required? 🤔

File: src/components/_modals/WithdrawModal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        size="lg"
+        backgroundColor="surface.tertiary"
+        flex={1}
+      >
+        {tabProps.children}
+      </SecondaryButton>
+    )
+  })
+  const id = useRouter().query.id as string
+  const buyUrl = strategyPageContentData[id]?.buyUrl
```
Comment (xanderjl) [inline src/components/_modals/WithdrawModal/index.tsx:51]: This component should be abstracted into its own sibling file

```
WidthdrawalModal/
- index.tsx
- CustomTab.tsx
```

**Review by xanderjl — CHANGES_REQUESTED** [review]
Just a few small tweaks and I'd say this is good to go 💪 

Excellent work @codingki

**Pull request merged**